app/controllers/passengers_controller.rb

```diff
@@ -42,7 +42,7 @@ def new
 
   def edit
     if @registrant.present? && !@registrant&.pending?
-      flash[:warning] = 'To edit your profile, please call (413) 545-2086'
+      flash[:warning] = "To edit your profile, please call #{I18n.t 'department.phone'}"
```

Comment:
```suggestion
      flash[:warning] = "To edit your profile, please call #{t 'department.phone'}"
```
`t` helper should be mixed in here I think?